lib/cartopy/mpl/geoaxes.py

```diff
@@ -999,12 +999,13 @@ def set_yticks(self, ticks, minor=False, crs=None):
 
         return super().set_yticks(yticks, minor=minor)
 
-    def stock_img(self, name='ne_shaded'):
+    def stock_img(self, name='ne_shaded', **kwargs):
         """
         Add a standard image to the map.
 
-        Currently, the only (and default) option is a downsampled version of
-        the Natural Earth shaded relief raster.
+        Currently, the only (and default) option for image is a downsampled
+        version of the Natural Earth shaded relief raster. Other options
+        (e.g., alpha) will be passed to func:`self.imshow`.
```

Comment:
```suggestion
        (e.g., alpha) will be passed to :func:`self.imshow`.
```
I think you need an extra colon in front of function to get that rendered properly by sphinx.